components/Marker/Marker.tsx

```diff
@@ -27,25 +35,28 @@ const Marker = ({
     >
       <Popup>
         <div>
-          <div className={styles.text}>
-            <h1>
-              {city},<br />
-              {country}
-            </h1>
-            {date && (
-              <i>
-                {getFullDateString(date)} ({getRelativeTimeString(date)})
-              </i>
-            )}
-            <br />
-            <span>{name}</span>
+          <div className={styles.background}>
+            <div className={styles.text}>
+              <h1 style={{ lineHeight: '1.2' }}>
+                {city},<br />
+                {country}
+              </h1>
+              <span style={{ fontWeight: '400' }}>
+                <i className="bi bi-calendar"></i> {getFullDateString(date)} (
+                {getRelativeTimeString(date)})
+              </span>
+              <br />
+              <span>
+                {getAuthorIcon()} {name}
+              </span>
+            </div>
           </div>
           <Image
             alt={`${name} at ${city}`}
             src={photo}
             layout="fill"
             objectFit="cover"
-            style={{ borderRadius: '10px', overflow: 'hidden' }}
+            style={{ borderRadius: '10px', overflow: 'hidden', zIndex: '0' }}
```

Comment:
👆

Reply:
And done ;) [1d15537](https://github.com/cesium/CeSIUMpeloMundo/pull/66/commits/1d15537b3e6bd83f4b2191df015cf9c427892442)